### PR TITLE
Fixing css validation issues

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
For a while sass validations errors were not showing or causing PRs to fail. This has meant a lot of code has gone into production which our linter isn't happy with. 

This fixes the code which doesn't abide by our linting rules which makes it so that we can start failing PRs on them again. ( @sndrs )

## What is the value of this and can you measure success?
Better code quality and consistency

## Does this affect other platforms - Amp, Apps, etc?
It affects amp styling, but not apps or any other off platform (only in the frontend repo)

@guardian/dotcom-platform

### CODE REVIEW

File: static/src/stylesheets/module/commercial/glabs/_hosted-gallery.scss [hunk 1]
```diff
@@ -548,12 +548,8 @@ $progressColor: #ffffff;
     width: $progressHeight;
 
     &:before {
-        content: "";
+        content: '';
         position: absolute;
-        background-image: -o-radial-gradient(center center, ellipse cover, $neutral-1 0%, transparent 50%);
```
Comment (NataliaLKB) [inline static/src/stylesheets/module/commercial/glabs/_hosted-gallery.scss]: @lps88 I have removed these as I think auto prefixer deals with this anyway. Is that okay with you?

File: static/src/stylesheets/module/facia/_item.scss [hunk 2]
```diff
@@ -157,7 +157,6 @@ $fc-item-gutter: $gs-gutter / 4;
 
     a {
         color: $neutral-1;
-        color: inherit;
```
Comment (gtrufitt) [inline static/src/stylesheets/module/facia/_item.scss]: Isn't the `$neutral-1` a fallback for browsers that don't support `inherit`?

**Review by gtrufitt — APPROVED** [review]
1 minor query but LGTM - yay for linting!

File: static/src/stylesheets/module/facia/_item.scss (hunk 2 above)
Comment (NataliaLKB) [inline static/src/stylesheets/module/facia/_item.scss]: didn't know that! But it does make sense. I will put this line back.

**Pull request merged**